docs/DeveloperManuals/Project.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Project"
+sidebar_position: 5
+description: >
+  `Project` is an object that can be used for DORA to associate `changes(commits)`, with `deployments` and `issues`.
+---
+
+## Summary
+`Project` is an object that can be used for DORA to associate `changes(commits)`, with `deployments` and `issues`.
```

Comment:
Do not present `project` as some kind of subsystem for DORA, no.
Lets say it is **a set of scopes from different domains**, a way to group different resources, and it is crucial for some metric calculation like DORA.